web/html/src/components/input/Form.js

```diff
@@ -100,7 +104,20 @@ export class Form extends React.Component<Props> {
 
   componentDidUpdate(prevProps: Object) {
     if (prevProps.model !== this.props.model) {
-      Object.keys(this.inputs).forEach(name => this.inputs[name].validate(this.props.model[name]));
+      Object.keys(this.inputs).forEach(name => {
+        const names = name.split(',');
```

Comment:
If we want to go with the merged string route (please see other comments for discussion), I think it would be good to add separate methods to get the canonical names, essentially just wrappers around split and join. The reason for this is that it's very easy to misplace a `split()` vs `split(',')`, especially if you see `join()` and don't work with this code often.

Reply:
sounds good, but these are only internal to the `Form` component... is it worth it?

Reply:
done